doc/tutorial/climada_entity_Exposures_osm.ipynb

```diff
@@ -0,0 +1,692 @@
lFh+yRasVZOTm/rjmdKZN9l5EfBUq8R4F3Fh5czZSuYwBJceI4U3j9wxl84Q9hPiF6AXXmFmZ6liV/UxPw6AVPWiazhKurqKNeIwbny+md+GRWdpPZviZmYqMcM3lTLNB9Xlz7wOX994Ql5dvXIoyDYuzd84Rgj+HYmJuzPHonTQy74hUpmsVmlq1rNauIr3nddSrUtc3a0+yAVmDV1FP1j9Os+Vb1nc9pzZe0FnHyEyBk7sDq+7No3PuAcSE6js4FaqUD4lUgt/Lr0SHxdB5Smu6TGtj8HwjQ6Jo7dyLSi3LUrl5WWp0qMzdUw/ZNmUfH1Ib0NNVb+ecnkiJWoW5uOs6Vw7com63asxqq0voXb2cCPDVdeUxtTAhWy4XfJ5+omGf2oIsJANUSSrmdFjOzSN3cc3ljIObvVhVBKH/omyDEgbPIz1un3jA4l5riQqJpk63agxd2wcjYwXR4TFsGLWTs9suU7NTFe6ceIA6WU1inBKJVELnqa35Y8kJ4qMSMm0uz1UsB+sepUkVTm04j0wuo1LLssxovYhXd96jjNUdtKVvZs5Tyov4qHi+vdeXGUCaTl2mkFKkSkGDjebpZ4UMXesuORxpNaoJdbpWBSSYmpuwbuR2Di1NmxE9o9qHRCLh4q7rLO61ViSsdq42zDoxXscNJNW+s0aHSnSb2Y4uuQbpHVOtLlW4sOMaNg5WbH23wmDQjkajYWmf9ZRpUOI/ZmX4IwR/CeXoyj/pMacDcrkwoNdqtTy98pLDy05RuVU5suVyoWAF7//K8aR36Oo9vxNtRjfN8rbRYTHM6bhcZ5A5/9xkStQyXMQAuLDrGvO7rASgWI1CLLwgJDEnxCbQyqknSYkqll6fSaGK+TLdB8CC7quQSCTkLJSdViMa67wWFRpNq5RMlSYD64oJyEuvz6BQxfxEhkQRERRFzkxSq/8qzmy5xLJ+G1h6fSb5y/764Knf+Dn8bk79ByI6LIbx9Wfz/qFPpmmEhqz2sgIrewuiw2KRyqUYGSv0/NOL1SjE/HOTM20IjAqN5uzWyzy88Ax1spqnV14arFBJZVJccjqREJvIAf+NhH4LY+WgTbx79JHQL7rkucXwhuQunpNVQ7YQnyK9MLc1o9WIxnQY30LnWO6decyFnVfxKpKDrZP2ilVrCxsz4qIT0Gq02LvZsv39KuQKGVqNltCAcC5sv6qTaGqfzZZdn9Ygk8lEd5Hdsw8RGRzFlQO3sLAyIyI4inlnJzK43ERxu+lHxjC1eZr3cfX2FYmPTuDuqUfYOFszbG0flg/cSETK4MLYzJhjkduRyWWEfgujvYdgvWbrYk1EYBQKUzlqlVrw/VapUSdryFkkOx+f6c9+ZIbhG/uytPd67FxtsLAxZ/PLZeJrXfMOwj/FlrP16CYcXJgWUFO7axXOb0+Tc8jkUtFaEAxb5RmKpv8e0su6Ugc+N47dY3rzhZlus+3dCi7vvcnuWX9gZKIgPi7RYEPsTt/VuORw0lnWxLYTCVHpvLRTyNe4nUOo2bGyuDwuKo6F3Vdz8+h9ne3LNixBQlwiz9I1EBubGWPjaEWQXwg2TlY6TilNB9Vj0Ar9cLJUPLn8gtBv4VRrWwG5Qo4qScX8LitRJqh4euUFyUlqsf9EPOYUL3RbZ2us7C2IDI7OVEbkksORUVsGEhMRy/SWi/DwzsaW18sNrjuq5jSeXn6ps0xhrGDumYkUrVow03PIiJiIWGIj4nD1ctZ77eXtt4ytMxN1UrJYyW83rhk953QUPo8rL5nfZQUqpYqY8Djq96hBfHQ8l/ffwsbJioOBm/n0+jP9S40hOcHw/c3MypT4mIQfux2lDIrc82bj6zt/nZcyemSnwrOgB34vv2TqMpPq2FOrcxXGbk+ToB1Z8SdrhgkDfit7C3rO7USDXjVJUqr4+tafr+/8iY9JwCWHE36vvlK4cn40ag39SwqypmmHR1OxWRl6FR6O30tBlnZGtY+Lu66xeeIewv0jxfdycLNjwLLuVGpR9j8W8vVXsKz/Bk6tF5J8UwcX/ZZ2ZXm/jaiUKkrXK063mW31EmP/kxheZbJoMtBxUks6T2n9Uw2n6mQ1rV16ifaNm14uFXvB0iM1VA6EGcTD4dt0+jPaZutNeGAkK27PJn/ZvFl634TYRL1BmjpZTT0joYJvbm1GiVpF8H32CXWyhg1PF7Fl4l5OrT9Pr3mdaDGsoc628TEJyORS4qMTRLnOp1ef6VtsNJpkDR0mtaDZoAaZyorePfQhd/Gcf8ss4Dd+DX4T938ods36gz2zD+Ho7mCwUeWvIKMFWipsnKyp2KI0p9YJFcjxu4dQo31lvfUMYe+8I2yZsMfga/bZbFl+azZh38I5uPgEdi42fHzxWbTJS8WhkC2sHraVS+kaCduNbUaPOR30Hkzp0+ZG157OkxTXiqHr+7Bl/G4hFEcqPDiaDa7Plb03KVO/OE7ZHdk7/zBJCckkxCQwbtdganbQbY5satMFKztLAj+lVTSXXJ3BnI7L9BJUQaiET94/gsoty3HrxH2W99tIQlwCCdFphKD9xOb0mCnc0N898GFgmXGGP8i/arsogbWPFtK/eFpzYnrN9om1Z1kxUN8XvmyDEvRZ1AVnTwcUxgokEglJiUk6/Qwn4nYxv8tKbhy6Ky7Liq1gRsgUMmafHM+BRcdp1LcOS3qtJTbyx3kEdtlsdAiLh3c2Qv3DSYhJZNDKnjQdWE9n/T+WnGB9io+51EiKJkmDVCZl0r7hVG5ZzuB7qJPVtHDorhfGkr9cHpKT1TQfXJ9anaqiUWu4dfwBl/Ze58ahu0jlUjTJGqq2rcCkvcN/6vNIRVhQBEPKTSD4OyFXphYmdJ7WhqMr/hR7U9qMaoKNkzVeRT0pVqMQMpmM2Mg4Wjr2QKPWsPvTGqJCY9g6eR9l6henyYC6fH79jd6FR4j7rda2AsWqF6JGx8p6CZ4fX36mT8rMTLuxzegyvU2WNdOPLz9ndrulxEXFk5ykxsHdjt2
```

Comment:
I think the default example should not be the pre-defined infrastructures, but the free osm keys. The infrastructure case is a special case, and the osm keys is the normal case.